### PR TITLE
WMCO 2.0.4 release notes

### PR DESCRIPTION
**Pull request by codyhoag**
Preview: https://deploy-preview-40978--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-2-x.html#wmco-2-0-4

### CODE REVIEW

**Comment by codyhoag** [comment]
@rrasouli can you verify? Thanks!

File: windows_containers/windows-containers-release-notes-2-x.adoc [hunk 1]
```diff
@@ -19,6 +19,20 @@ You must have a subscription to receive support for the Red Hat WMCO. Deploying
...
+== Release notes for Red Hat Windows Machine Config Operator 2.0.4
+
+Issued: 2022-02-14
+
+The WMCO 2.0.4 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHSA-2022:0241[RHSA-2022:0241].
+
+[id="wmco-2-0-4-bug-fixes"]
+=== Bug fixes
+
+* A flaw was found in the way Unicode standards are implemented in the context of development environments, which have specialized requirements for rendering text. An attacker could exploit this to deceive a human reviewer by creating a malicious patch containing well placed BiDi characters. The special handling and rendering of those characters could have been used to hide unexpected and potentially dangerous behavior from the reviewer. This security issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2005819[**BZ#2005819**])
```
Comment (ctauchen) [inline windows_containers/windows-containers-release-notes-2-x.adoc:32]: A flaw was found in the way Unicode standards are implemented in the context of development environments, which have specialized requirements for rendering text.

"that have specialized"?? Unless it's all development environments.

Maybe: s/BiDi/bidirectional

s/well placed/well-placed per https://www.merriam-webster.com/dictionary/well-placed

Comment (ctauchen) [reply windows_containers/windows-containers-release-notes-2-x.adoc:32]: Should this have the same BZ number as the WMCO 1.0.6 release? Is one of these a backport with a different number?

**Review by ctauchen — COMMENTED** [review]
A few things, otherwise LGTM.

Not familiar with this component -- is this really only for 4.7?

**Comment by codyhoag** [comment]
@ctauchen this specific BZ has been attached to many erratas across products. I'm not sure what the backstory is, but the same BZ was attached to both WMCO advisories.

**Pull request merged**